scripts/sentiment_analysis/process_data.py

```diff
@@ -86,7 +86,7 @@ def _build_vocab(data_name, train_dataset, test_dataset):
         test_dataset[i][0] = _clean_str(line[0], data_name)
         line = test_dataset[i][0].split()
         max_len = max_len if max_len > len(line) else len(line)
-    all_token.extend(line)
+        all_token.extend(line)
```

Comment:
@xiaotinghe could you explain this change? Did I change it by accident or was there a bug before? Thanks for checking the changes!

Reply:
There was a bug before.
I happened to see it.

Reply:
Will this affect the results in logs?

Reply:
@xiaotinghe the CI fails since your change, can you make it pass again?

Reply:
@leezu there was a problem with nbsphinx version that @eric-haibin-lin fixed just now. Try doing and rebase and see if the issue persists